python/kvikio/kvikio/__init__.py

```diff
@@ -16,6 +16,7 @@
 from kvikio._version import __git_commit__, __version__
 from kvikio.cufile import CuFile, clear_page_cache, get_page_cache_info
 from kvikio.mmap import Mmap
+from kvikio.remote_file import RemoteEndpointType  # noqa: F401
```

Comment:
Add `RemoteEndpointType to ```__all__``` and you don't need the noqa
```suggestion
from kvikio.remote_file import RemoteEndpointType
```